### ISSUE TITLE
About Hyperparameter

### ISSUE THREAD

**Issue by zhouwy19** [dunbar12138/pix2pix3D#18]
Thank you for your great work.

I would like to inquire about the settings of hyperparameters.

1. Which type of seg_weight do you use?(https://github.com/dunbar12138/pix2pix3D/blob/63dc8d9535ed853e12aeed725b505be6755b8a54/training/loss.py#L338)
2. What are the learning rates of the G, D, D_semantic and Encoder？

Thank you.

**Comment by dunbar12138** [comment]
Hi, thanks for your interest. We use `seg_weight=2.`  We used 0.002 as the learning rate, following the EG3D codebase.
